jobs/email/EditorialReminder.php

```diff
@@ -81,7 +81,7 @@ public function handle(): void
 
         /** @var int $submissionId */
         foreach ($submissionIds as $submissionId) {
-            $submission = Repo::submission()->get($submissionId);
+            $submission = Repo::submission()->get((int)$submissionId);
```

Comment:
Is this change necessary?

Reply:
@asmecher this is added because of this error posted in forum https://forum.pkp.sfu.ca/t/ojs-3-4-0-1-failed-jobs/80170/4?u=touhidur . I also could not reproduce it in my testing but may be this is one of those platform specific type issues that we are experiencing too randomly .

Reply:
I think this is only happening because you've got [strict types](https://github.com/pkp/pkp-lib/pull/9213/files#diff-83d61d197c61d91e16b32cc318edb9f2c3ba21032384d2337d75ce12c7189fdfR3) enabled on the file; otherwise the `string` would be automatically coerced to an `int` when passed into the submission repository's `get` function.

We don't yet have a set of development patterns that will gracefully allow us to adopt strict typing -- so rather than peppering the code with manual typecasts when errors arise, I'd rather just disable strict typing.